.circleci/config.yml

```diff
@@ -19,58 +20,48 @@ push_builds_only: &push_builds_only
 executors:
   amphtml-docker-small-executor:
     docker:
-      - image: cimg/base:2021.05
+      - image: cimg/base:stable
     resource_class: small
   amphtml-medium-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
     resource_class: medium
   amphtml-large-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
     resource_class: large
   amphtml-xlarge-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
+    resource_class: xlarge
+  validator-xlarge-executor:
+    docker:
+      - image: cimg/openjdk:16.0-node
     resource_class: xlarge
 
 commands:
-  restore_workspace:
+  setup_vm:
     steps:
       - attach_workspace:
           at: /tmp
       - run:
-          name: 'Rename Restored Workspace'
+          name: 'Rename Restored Workspace and Create New Workspace'
```

Comment:
nit: Similar suggestion: Maybe shorten the step name to "Setup Workspace"?

Reply:
And for this one, "Configure Workspace"?

Reply:
`Configure Temporary Workspace`